python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -1341,7 +1341,7 @@ def __init__(self):
         ([2, 8], torch.float32, True),
     ])
     def forward(self, x):
-        return torch.broadcast_to(x, [1, 2, 8])
+        return torch.broadcast_to(x, [2, 2, 8])
```

Comment:
nit: Instead of changing this test, can you please create a new one? In this way, it looks like we are not intending to support `[1, 2, 8]` anymore.

Reply:
Done!